tests/frontend/correspondence_generator/keypoint_aggregator/test_keypoint_aggregator.py

```diff
@@ -0,0 +1,120 @@
+"""Unit tests for keypoint aggregation for correspondence generators.
```

Comment:
Major comments:
1. Why not define the input data as common?
2. Test on floating point input?

Minor:
1. Two different files and test classes?

Reply:
Updated to float, and added base class test file, and two more test files.